metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery_schema_gen.py

```diff
@@ -676,10 +690,92 @@ def _process_snapshot(
             dataset_name=dataset_name,
         )
 
-    def make_tag_from_label(self, key: str, value: str) -> str:
-        if not value.startswith(ENCODED_TAG_PREFIX):
+    def make_tag_urn_from_label(self, key: str, value: str) -> str:
+        if value:
             return make_tag_urn(f"""{key}:{value}""")
-        return self.modified_base32decode(value)
+        else:
+            return make_tag_urn(key)
+
+    def get_platform_resource(self, primary_key: str) -> Optional[PlatformResource]:
+        # if graph is not available we always create a new PlatformResource
+        if not self.graph:
```

Comment:
My thinking was, what if I wanted to write aspects to the file and load them to DataHub from the file? 
In this case, even though this will always recreate the platform resources, it will not fail, and you can use it in an air-gapped environment.